spec/rubocop/cop/lint/empty_when_spec.rb

```diff
@@ -47,7 +63,13 @@
         else 3
         end
       RUBY
-      expect_no_corrections
+
+      expect_correction(<<~RUBY)
+        case foo
```

Comment:
This is not equivalent at all...
I'm not sure that even when equivalent it's such a good idea to do so.

Reply:
😅 Correct, my bad. Updated code and tests.

When there is an empty `when` clause it is probably that developer made some refactoring and forgot to remove that branch. So it is a good idea to suggest and when it is equivalent, it is not a (at least 'very') bad idea to autocorrect for him, like other `Empty...` cops do.